ocs_ci/cleanup/vsphere/cleanup.py

```diff
@@ -123,14 +149,37 @@ def delete_ips(self, cluster_name):
             node_fqdn = f"{node}.{cluster_name}.{config.ENV_DATA['base_domain']}"
             all_nodes.append(node_fqdn)
 
-        logger.info(f"Removing IP's for nodes {all_nodes} from IPAM server")
+        for node in all_nodes:
+            self.delete(node)
+
+    def delete_ips_ipi(self, cluster_name):
+        """
+        Delete IP's from IPAM server for IPI cluster
+
+        Args:
+            cluster_name (str): Name of the cluster to release IP's
+                from IPAM server
+
+        """
+        for i in range(0, 2):
```

Comment:
can there be more than 3 IPs to delete? What if there are more than 3 nodes?

Reply:
those are the API and ingress IP's ( not the node IP's ). Its always 2 IP's

Reply:
ack